ClubManagement/ClubManagement/Activities/Base/CreateOrEditEvent.cs

```diff
@@ -0,0 +1,305 @@
+﻿using System;
+
+using Android.App;
+using Android.Content;
+using Android.OS;
+using Android.Views;
+using Android.Widget;
+using Android.Gms.Location.Places;
+using ClubManagement.Ultilities;
+using Android.Gms.Location.Places.UI;
+using ClubManagement.Fragments;
+using ClubManagement.Controllers;
+using ClubManagement.Models;
+
+namespace ClubManagement.Activities.Base
+{
+    public class CreateOrEditEvent
```

Comment:
It need to be an Activity, Fragment, Dialog or something else

Reply:
Done